unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Events/PointerInputRepresentantion.cs

```diff
@@ -53,21 +53,21 @@ public void SetData(IParcelScene scene, IDCLEntity entity, bool showFeedback, in
 
         public void Dispose()
         {
-            dataStore.shapesReady.OnAdded -= ConfigureColliders;
+            dataStore.shapesReady.OnAdded -= CreateColliders;
             pointerEventHandler.Dispose();
         }
 
         public Transform GetTransform() { return eventEntity.gameObject.transform; }
-        
+
         IDCLEntity IPointerEvent.entity => eventEntity;
-        
+
         void IPointerEvent.SetHoverState(bool hoverState)
         {
-            pointerEventHandler.SetFeedbackState(showFeedback, hoverState, button.ToString(), hoverText);
+            pointerEventHandler.SetFeedbackState(showFeedback, hoverState, ((WebInterface.ACTION_BUTTON) button).ToString(), hoverText);
```

Comment:
((WebInterface.ACTION_BUTTON) button).ToString() <-- This will dissapear when we implement the enum for the buttons